cea/optimization/prices.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""
+Data required for Slave from Master
+This File sets all variables for the slave optimization, that have to be set by the Master
```

Comment:
this comment doesn't say anything. the file is called `prices`. maybe talk about that instead?